components/Upcoming.js

```diff
@@ -1,7 +1,12 @@
 import React from "react";
```

Comment:
Delete any changes from this file, this file will be implemented in another story

Reply:
what should I do?